README.md

```diff
@@ -0,0 +1,23 @@
+# GPSOAuth
```

Comment:
I think I'd rather leave it as is for consistency's sake with other references (ports, issues, etc).

Reply:
Sure, reverted this.